README.md

```diff
@@ -8,7 +8,7 @@ The `rwahs` command handles all steps of integration and deployment via various
 
 The general syntax is:
 
-    rwahs <task> -e <environment> -c <component>...
```

Comment:
I decided that I'm not supporting multiple components in one call, it makes things complicated.  If we find we are doing the same sequence of steps each time, we can just create a simple wrapper script.